src/styles/components/breadcrumbs.scss

```diff
@@ -25,6 +25,55 @@
 
 @media screen and (max-width: 1024px) {
 	.breadcrumbs {
-		display: none;
+		display: inline-block;
+		text-align: left;
+		margin: auto 0;
+		h4 {
+			font-size: 20px;
```

Comment:
@Aringoaway Why did you change the font-size here from 24 to 20px?

Reply:
@Pe5h4 because the font size is too large for a small screen